packages/remix-dev/config/defaults/entry.server.cloudflare.tsx

```diff
@@ -22,7 +22,7 @@ export default async function handleRequest(
     }
   );
 
-  if (isbot(request.headers.get("user-agent"))) {
+  if (isBotRequest(request.headers.get("user-agent"))) {
```

Comment:
Internal implementations of `entry.server.tsx` will support either v3 or v4 for backward compatibility with existing Remix apps who have pinned v3 in their package.json